README.md

```diff
@@ -1,9 +1,176 @@
-# telestion-extension-template
+# telestion-extension-mavlink
 
-A template for creating new Telestion extensions.
+A Mavlink message parser for the Telestion backend.
 
-This template contains an initialized project structure with configured dependencies and a publishing pipeline.
+## Install
 
-To fetch the dependencies locally you need a Github personal access token with `packages:read` rights.
-Either add your github user name and the token as `GITHUB_ACTOR` and `GITHUB_TOKEN` to your environment variables 
-or fill the `.github.credentials.template`.
+First, add the extension package registry to your gradle repositories:
+
+```groovy
+repositories {
+    maven {
+        name = "GitHubMavlink"
+        url = uri("https://maven.pkg.github.com/wuespace/telestion-extension-mavlink/")
+        credentials {
+            username = System.getenv("GITHUB_ACTOR") == null ? GITHUB_ACTOR : System.getenv("GITHUB_ACTOR")
+            password = System.getenv("GITHUB_TOKEN") == null ? GITHUB_TOKEN : System.getenv("GITHUB_TOKEN")
+        }
+    }
+}
+```
+
+Next, add the extension to the project dependency list:
+
+```groovy
+dependencies {
+    // ...
+    implementation 'de.wuespace.telestion.extension:mavlink:0.4.0'
+    // ...
+}
+```
+
+And synchronize gradle.
+
+Finished!
+
+## Usage
+
+The extension provides different verticles, that you can use and set up in your project `config.json`.
+You will need:
+
+- [Mavlink v2 Validator](https://wuespace.github.io/telestion-extension-mavlink/de/wuespace/telestion/extension/mavlink/ValidatorMavlink2.html)
+- [Mavlink Message Registrar](https://wuespace.github.io/telestion-extension-mavlink/de/wuespace/telestion/extension/mavlink/MavlinkRegistrar.html)
+- [Payload Parser](https://wuespace.github.io/telestion-extension-mavlink/de/wuespace/telestion/extension/mavlink/PayloadParser.html)
+
+The `Mavlink Validator` verticles for `Mavlink v1` and `Mavlink v2` validate the incoming encoded mavlink message
+and sends it to the parser on a valid mavlink message.
+Additionally, all encoded messages are sent to the `Mavlink Logger`
```

Comment:
This only works if the logger was also specifically specified in the config.json.

Reply:
Done in [`005a9f0` (#37)](https://github.com/wuespace/telestion-extension-mavlink/pull/37/commits/005a9f048a9dea28c4a7fafb865e05fd65f6cfa3)